### PR TITLE
API provides functionality to send articles in different languages

### PR DESCRIPTION
**Pull request by DavDan1**
PT: https://www.pivotaltracker.com/story/show/178341620
_____
### User Story
```
As a user
To understand articles on this website
I wand to see articles in my language of choice
```
____
### Changes Proposed
- Add new attribute Language to Article model 
- Check params: if lang=en/se -> send English/Swedish Articles
- Change seed
- Change factory
- Make English default
____
### What we learned
- Differens between body and route params in rspec

### CODE REVIEW

**Review by arunbhalli — APPROVED** [review]

File: app/serializers/articles_index_serializer.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class ArticlesIndexSerializer < ActiveModel::Serializer
-  attributes :id, :title, :teaser, :date, :category, :image, :rating, :author, :premium, :published, :comments
+  attributes :id, :title, :teaser, :date, :category, :language, :image, :rating, :author, :premium, :published,
+             :comments
```
Comment (1CIM) [inline app/serializers/articles_index_serializer.rb:3]: lol what happened here? 😅

**Review by 1CIM — APPROVED** [review]
For what i can see it look nice :3 GJ boiis ❤️

File: app/serializers/articles_index_serializer.rb (hunk 1 above)
Comment (1CIM) [reply app/serializers/articles_index_serializer.rb:3]: lel looks strange only in the file changes for me 🤷‍♂️

Comment (DavDan1) [reply app/serializers/articles_index_serializer.rb:3]: magic

**Pull request merged**